test/JUnit.Xml.TestLogger.AcceptanceTests/JUnitTestLoggerAcceptanceTests.cs

```diff
@@ -46,11 +46,6 @@ public void TestResultFileShouldContainTestSuitesInformation()
             var node = this.resultsXml.XPathSelectElement("/testsuites");
 
             Assert.IsNotNull(node);
-            Assert.AreEqual("JUnit.Xml.TestLogger.NetCore.Tests.dll", node.Attribute(XName.Get("name")).Value);
-            Assert.AreEqual("52", node.Attribute(XName.Get("tests")).Value);
```

Comment:
Aren't these validations relevant?

Reply:
No, Those elements aren't actually in the xml schema for junit, so they are removed in the logger class and here. None of the other changes I made so that the `LoggedXmlValidatesAgainstXsdSchema` test passes, were not things I had asserts for. 

I did test this in GitLab, and their parser didn't have any problems with it.